### DISCUSSION TITLE
Bithermical swich

### DISCUSSION THREAD

**Discussion by Yocee84** [Beormund/Tasmota32-Multi-Zone-Heating-Controller#12] [Ideas]
Hi!

I congratulate this super project!

I need this like a piece of bred!

But. I need a little plus thing. One outside thermometer (ds18****) measure the temperature. When i set one value (0 celsius), below start gas boiler (relay1), above start Heatpump (relay2).
Unfortunatly im stupid for made this, you my only hope!

**Comment by Beormund** [comment]
Sorry I don't understand your question.

**Reply by Yocee84** [reply]
Can you made this option? I set one temperature setpoint ( 0 celsius outside) Belove start relay 1, above start relay 2 when heating signal came from thermostats.

**Reply by Bournetoride** [reply]
I think Yocee is asking for the same as me, some method of bringing temperature value from ESP32 into Berry code

**Comment by Yocee84** [comment]
![Névtelen](https://user-images.githubusercontent.com/25384303/216003194-fa37ccab-5446-4d98-8f39-82c1cd4e51de.png)
